### PR TITLE
Fixes #112 App crash fixed

### LINKED ISSUE #112: CursorIndexOutOfBoundsException at the time of App launch.
**Actual Behaviour**

When we try to open the app on phones having photos less than 21, the app crashes with CursorIndexOutOfBoundsException.

**Expected Behaviour**

The app should not crash.

**Steps to reproduce it**

Try to run the app on an emulator with no photos.

**Would you like to work on the issue?**

Yes. Already worked on it. Sending the PR.

### PR DESCRIPTION
**Pull request by anantprsd5**
Fixes issue #112 App crash on phones with less photos fixed.

### CODE REVIEW

File: app/src/main/java/vn/mbm/phimp/me/newGallery.java [hunk 1]
```diff
@@ -3535,7 +3536,12 @@ public static void resumeLocalPhoto(int resum_number){
                     statsCounted = true;
                 }
 
-				for (int i = (turnsDone * PER_TURN); i < localImagesPerTurn; i++) {
+                if(localImageCount<localImagesPerTurn)
+                    finalImageCount = localImageCount;
+                else
+                    finalImageCount = localImagesPerTurn;
+
+				for (int i = (turnsDone * PER_TURN); i < finalImageCount; i++) {
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/newGallery.java]: Fix the indentation here.

File: app/src/main/java/vn/mbm/phimp/me/newGallery.java [hunk 2]
```diff
@@ -3535,7 +3536,12 @@ public static void resumeLocalPhoto(int resum_number){
                     statsCounted = true;
                 }
 
-				for (int i = (turnsDone * PER_TURN); i < localImagesPerTurn; i++) {
+                if(localImageCount<localImagesPerTurn)
+                    finalImageCount = localImageCount;
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/newGallery.java]: Single line conditional statements should also be inside braces.

File: app/src/main/java/vn/mbm/phimp/me/newGallery.java (hunk 1 above)
Comment (anantprsd5) [inline app/src/main/java/vn/mbm/phimp/me/newGallery.java]: Sure.

**Comment by anantprsd5** [comment]
@pa1pal The indentation was already like that, thats why I had to change so many lines.

**Pull request merged**